core/templates/services/platform-feature.service.ts

```diff
@@ -167,10 +167,12 @@ export class PlatformFeatureService {
       PlatformFeatureService.featureStatusSummary = await this
         .loadFeatureFlagsFromServer();
       this.saveResults();
-    } catch (err) {
-      this.loggerService.error(
-        'Error during initialization of PlatformFeatureService: ' +
-        `${err.message ? err.message : err}`);
+    } catch (err: unknown) {
+      if (err instanceof Error) {
+        this.loggerService.error(
+          'Error during initialization of PlatformFeatureService: ' +
+          `${err.message ? err.message : err}`);
```

Comment:
Out of curiosity, any reason not to use ``err.message || err`` instead?

Reply:
I just copied this code without going throught it, you solution makes more sense.